device-types/Cisco/N7K-C7009.yaml

```diff
@@ -0,0 +1,55 @@
+---
+manufacturer: Cisco
+model: N7K-C7009
+slug: cisco-n7k-c7009
+part_number: N7K-C7009
+u_height: 14
+weight: 100
+weight_unit: lb
+is_full_depth: true
+airflow: right-to-left
+subdevice_role: parent
+comments: '[Cisco Nexus 7000 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/nexus-7000-series-switches/Data_Sheet_C78-437762.html)'
+module-bays:
```

Comment:
Since this a parent device, are there any devices that slot into this? If so, then you should have all device slots be under `device-bays` rather than `module-bays` which are reserved for things like PSUs, Fans, SFP interface cards, etc. I dont use these so I do not know, but it looks like this should have device bays

Reply:
Nexus 7009 is only module bays, there should be no child devices really